dotnet/samples/KernelSyntaxExamples/Example99_ApiManifest.cs

```diff
@@ -0,0 +1,170 @@
+        {
+            this.WriteLine("Azure endpoint, apiKey, deploymentName, or modelId not found. Skipping example.");
+            return;
+        }
+
+        s_kernel = Kernel.CreateBuilder()
+            .AddAzureOpenAIChatCompletion(
+                deploymentName: chatDeploymentName,
+                endpoint: endpoint,
+                serviceId: "AzureOpenAIChat",
+                apiKey: apiKey,
+                modelId: chatModelId)
+            .Build();
+    }
+
+    private void WriteSampleHeadingToConsole(string goal, string expectedOutputDescription, params string[] pluginNames)
+    {
+        this.WriteLine();
+        this.WriteLine($"======== [ApiManifest Plugins] Create and Execute \"{goal}\" Plan ========");
+        this.WriteLine($"======== Plugins: {string.Join(" ", pluginNames)} ========");
+        this.WriteLine($"======== Expected Output: {expectedOutputDescription} ========");
+        this.WriteLine();
+    }
+
+    [Theory]
+    [InlineData("show the subject of my first message", "latest email message subject is shown", "MessagesPlugin")]
+    [InlineData("get contents of file with id=test.txt", "test.txt file is fetched and the content is shown", "DriveItemPlugin", "MessagesPlugin")]
+    [InlineData("get contents of file with id=test.txt", "test.txt file is not fetched because DriveItemPlugin is not loaded", "MessagesPlugin")]
+    [InlineData("tell me how many contacts I have", "number of contacts is shown", "MessagesPlugin", "ContactsPlugin")]
+    [InlineData("tell me title of first event in my calendar", "title of first event from calendar is shown if exists", "MessagesPlugin", "CalendarPlugin")]
+    public async Task RunSampleWithPlannerAsync(string goal, string expectedOutputDescription, params string[] pluginNames)
+    {
+        _ = s_kernel ?? throw new KernelException("Kernel not initialized!");
+        WriteSampleHeadingToConsole(goal, expectedOutputDescription, pluginNames);
+        s_kernel.Plugins.Clear();
+        await AddApiManifestPluginsAsync(pluginNames);
+
+        var result = await s_planner.ExecuteAsync(s_kernel, goal);
+
+        this.WriteLine("--------------------");
+        this.WriteLine($"\nResult:\n{result.FinalAnswer}\n");
+        this.WriteLine("--------------------");
+    }
+
+    private async Task AddApiManifestPluginsAsync(params string[] pluginNames)
+    {
+        _ = s_kernel ?? throw new KernelException("Kernel not initialized!");
+#pragma warning disable SKEXP0053
+        if (TestConfiguration.MSGraph.Scopes == null)
+        {
+            throw new InvalidOperationException("Missing Scopes configuration for Microsoft Graph API.");
+        }
+
+        string? currentAssemblyDirectory = Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location);
+        if (string.IsNullOrWhiteSpace(currentAssemblyDirectory))
+        {
+            throw new InvalidOperationException("Unable to determine current assembly directory.");
+        }
```

Comment:
All of our examples load files from embedded assembly resources; for example - [Example22_OpenAIPlugin_AzureKeyVault](https://github.com/microsoft/semantic-kernel/blob/b2bbd410c07739767b5aad48961a8ced5b4cf4a4/dotnet/samples/KernelSyntaxExamples/Example22_OpenAIPlugin_AzureKeyVault.cs#L70). It would be good to stick to one way of doing the same things.